### PR TITLE
docs: pass explicit 'client' in '{Blob.Bucket}.from_string' examples

### PR DESCRIPTION
**Pull request by tseaver**
See #540.

### CODE REVIEW

File: google/cloud/storage/blob.py [hunk 1]
```diff
@@ -405,7 +405,7 @@ def from_string(cls, uri, client=None):
             >>> from google.cloud import storage
             >>> from google.cloud.storage.blob import Blob
             >>> client = storage.Client()
-            >>> blob = Blob.from_string("gs://bucket/object")
+            >>> blob = Blob.from_string("gs://bucket/object", client=client)
```
Comment (cojenco) [inline google/cloud/storage/blob.py:408]: Thanks for the change! Could we also update both parameter descriptions with a note to strongly encourage specifying a client?

Comment (tseaver) [reply google/cloud/storage/blob.py:408]: 745e531

**Review by cojenco — APPROVED** [review]
Thanks so much for the clarification!

**Comment by tseaver** [comment]
Flaky systest is #541.

**Pull request merged**